### ISSUE TITLE
doubles the number of references to when using "Find All References"

### ISSUE THREAD

**Issue by SHxKM** [microsoft/pyright#201]
**Describe the bug**
When pyright is enabled, "Find All References" finds x2 references for functions (I haven't tested other stuff).

**To Reproduce**
1. Install and Enable `pyright`.
2. Attempt to "Find All references" for a function
3. You'll get 2 results for the same reference, so a function referenced 3 times in 2 files will appear to be referenced 6 times in 2 files.

**Expected behavior**
I'll get exactly the number of references that I actually have in code.

**VS Code extension or command-line**
Extension.

**Comment by erictraut** [comment]
Thanks for the bug report. I'm not seeing this problem, so there must be something different about your configuration. By any chance, are you using a multi-root workspace?

**Comment by SHxKM** [comment]
@erictraut thanks for your fast reply and hard work.

> By any chance, are you using a multi-root workspace?

I am not. 

I also noticed that I'm getting a weird pop-up whenever I Option-click a definition when `pyright` is installed. That window too finds double the references. Example:

<img width="993" alt="derdemo" src="https://user-images.githubusercontent.com/6573474/63100074-f6dd6600-bf7e-11e9-8079-ed48bde6612d.png">

These "two" references naturally point to the same `def`..

**Comment by erictraut** [comment]
Ah, I see what's going on. You have both Pyright and the VSCode Python extensions installed. The VSCode Python extension both implement language features like "find references", "find definition", hover and type completion. VSCode is receiving answers from both of these extensions and presenting them both to you.

I've started a discussion with that team that provides the VSCode Python extension to discuss ways to address this. One proposal is to add a setting that allows you to disable the language server in the VSCode Python extension so you can still get all of the type-checking benefits of pyright without getting duplicate results or paying the overhead for two completely separate analysis passes.

See here for more details.
https://github.com/microsoft/pyright/issues/197#issuecomment-521652582

In the meantime, the only workaround is to 1) live with the duplicates for now, or 2) disable the VSCode Python extension or 3) disable the Pyright extension.

**Comment by SHxKM** [comment]
Got it. Weird that VSCode itself doesn't support customization of this setting. Thanks for your responsiveness.

**Comment by erictraut** [comment]
I just published version 1.0.56 of pyright that contains a new setting called "pyright.disableLanguageServices". If you enable this setting, all language service features other than over will be disabled within pyright.

I also got agreement from the Microsoft team that develops the VS Code Python extension to add a setting that will disable the language services in that extension. Once this is released, you will be able to disable one or the other (or continue to run them both).
